firmware/Reload Pro.cydsn/ui.c

```diff
@@ -85,7 +89,7 @@ const menudata set_readout_menu = {
 	"Choose value",
 	{
 		{"Set Current", {NULL, (void*)READOUT_CURRENT_SETPOINT, 0}},
-		{"Act. Current", {NULL, (void*)READOUT_CURRENT_USAGE, 0}},
+		{"Act Current", {NULL, (void*)READOUT_CURRENT_USAGE, 0}},

```

Comment:
Oops that wasn't meant to end up in this pull
That's just my preference for alignment onscreen.
Will revert this one :)